--- proxmoxlib.js	2024-11-25 17:30:44.841057513 +0100
+++ /usr/share/javascript/proxmox-widget-toolkit/proxmoxlib.js	2024-11-25 14:15:33.129709171 +0100
@@ -1103,13 +1103,6 @@
 	return Proxmox.Utils.render_cpu_usage(value, record.cpus);
     },
 
-    render_node_temp: function(record) {
-	   if (!record || !Ext.isNumeric(record.used) || !Ext.isNumeric(record.total)) {
-			return '-';
-		}
-	return record.used.toFixed(1) + '°C (crit: ' + record.total.toFixed(1) + '°C)';
-	  },
-
     render_node_size_usage: function(record) {
 	return Proxmox.Utils.render_size_usage(record.used, record.total);
     },
