--- pvemanagerlib.js	2024-11-25 13:33:09.000000000 +0100
+++ /usr/share/pve-manager/js/pvemanagerlib.js	2024-11-20 11:39:37.000000000 +0100
@@ -44976,24 +44976,6 @@
 	    colspan: 2,
 	    padding: '0 0 20 0',
 	},
-	
-	{
-	    itemId: 'cputemp',
-	    iconCls: 'fa fa-fw fa-thermometer-half',
-	    title: gettext('CPU temp'),
-	    valueField: 'cputemp',
-	    maxField: 'cputemp',
-	    renderer: Proxmox.Utils.render_node_temp,
-	},
-	{
-	    itemId: 'disktemp',
-	    iconCls: 'fa fa-fw fa-thermometer-half',
-	    title: gettext('DISK temp'),
-	    valueField: 'disktemp',
-	    maxField: 'disktemp',
-	    renderer: Proxmox.Utils.render_node_temp,
-	},
-	
 	{
 	    itemId: 'cpus',
 	    colspan: 2,
